addons/base/views.py

```diff
@@ -692,9 +691,9 @@ def addon_view_or_download_file(auth, path, provider, **kwargs):
     return addon_view_file(auth, node, file_node, version)
 
 
-def addon_view_or_download_quickfile(**kwargs):
+def addon_view_or_download_file_by_fid(**kwargs):
```

Comment:
We can remove this since the front-end will no longer use the file_redirect route